lib/bambora/rest/www_form_client.rb

```diff
@@ -19,8 +19,8 @@ def post(path:, body:)
         # did not work in testing so I am manually transforming the hash into query parameters here.
         parse_response_body(
           super(
-            path: "#{path}?#{Bambora::Builders::WWWFormParameters.new(body: body)}",
-            body: nil,
+            path: path,
+            body: Bambora::Builders::WWWFormParameters.new(body: body).to_s,
```

Comment:
Moved this to the body as per the example in the bambora docs. Although, it works in both places just fine.